src/DockWidgetTab.cpp

```diff
@@ -504,7 +508,25 @@ void CDockWidgetTab::onDetachActionTriggered()
 	d->startFloating(DraggingInactive);
 }
 
-} // namespace ads
 
+
+
+//============================================================================
+bool CDockWidgetTab::event(QEvent *e)
+{
+	#ifndef QT_NO_TOOLTIP
+	if (e->type() == QEvent::ToolTipChange)
+	{
+		const auto text = toolTip();
+		d->TitleLabel->setToolTip(text);
+	}
+	#endif
+	return QFrame::event(e);
```

Comment:
Please use Super::event(e) here instead of QFrame::event(e). Super is defined in the CDockWidgetHeader.